supersid/config/config.py

```diff
@@ -10,11 +10,11 @@
 
 ### [PARAMETERS] ###
 
-# Path to the file directory locating the raw csv data you wish to process.
-data_path = 'C:/Users/oscar/OneDrive/Desktop/temp'
+# Paths to the file directories locating the raw csv data you wish to process.
+data_path = ['C:/Users/oscar/Desktop/temp']
 
 # Output path of archive - archive will be generated within the specified directory.
-archive_path = 'C:/Users/oscar/OneDrive/Desktop/SuperSid/data'
+archive_path = 'C:/Users/oscar/Desktop/SuperSid/data'
```

Comment:
maybe I'm missing something here - is this on both machines? and why are you moving it out of OneDrive?

Reply:
So this is my local machine config (weirdly my desktop as two locations) and wished to make it more accessible, each machine has its own local branch which changes those though. Couldn't think of a better way to do it any suggestions?

Reply:
ah ok - so when you install this on the different machines, it changes this path?

Reply:
Yes, there is like a commit that always sits on top just changing these paths based on the os machine file structure.

Reply:
ah ok perfect. This is good for now - was just curious :)